cocotb/__init__.py

```diff
@@ -57,7 +57,17 @@
 if "SPHINX_BUILD" not in os.environ:
     logging.basicConfig()
     logging.setLoggerClass(SimBaseLog)
-    log = SimLog('cocotb.gpi')
+    log = SimLog('cocotb')
+    _default_log = logging.INFO
+    if "COCOTB_LOG_LEVEL" in os.environ:
+        level = os.getenv("COCOTB_LOG_LEVEL")
+        try:
+            _default_log = getattr(logging, level)
+        except AttributeError as e:
+            pass
```

Comment:
Just a thought - we may want to log an error message here, otherwise for example `export COCOTB_LOG_LEVEL=DEBUGG` would silently not do what you expect.

Could also be tidier by using default value:

```
level = os.getenv("COCOTB_LOG_LEVEL", "INFO")
try:
     _default_log = getattr(logging, level)
except AttributeError as e:
    log.error("Unable to set logging level %s" % level)
    _default_log = logging.INFO
```